crawler.py

```diff
@@ -191,10 +191,10 @@ def run_all(driver):
 os.environ['GCOV_PREFIX_STRIP'] = str(strip_count)
 
 # Returns absolute path for created temp file
-gcov_dir = tempfile.mkdtemp()
+gcov_dir = tempfile.TemporaryDirectory()
```

Comment:
We should use a context manager here, to make sure the directory is always deleted.
You can do something like:
```Python
with tempfile.TemporaryDirectory() as gcov_dir, tempfile.TemporaryDirectory() as jsvm_dir:
    ...
```